### PR TITLE
Firxes #381 add table with x arguments

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@Yuri05 Tests are failing for now because of SimModel.

File: src/OSPSuite.Core/Serialization/ParameterIdentificationExport/ParameterIdentificationExportSchemaConstants.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 {
    public static class ParameterIdentificationExportSchemaConstants
    {
-      public const string Namespace = "http://www.systems-biology.com";
+      public const string Namespace = "http://www.open-systems-pharmacology.org";
```
Comment (msevestre) [inline src/OSPSuite.Core/Serialization/ParameterIdentificationExport/ParameterIdentificationExportSchemaConstants.cs]: Change namespace

File: src/OSPSuite.Core/Serialization/SimModel/DTO/FormulaExport.cs [hunk 2]
```diff
@@ -18,7 +18,7 @@ public ExplicitFormulaExport()
       public IDictionary<string, int> ReferenceList { get; private set; }
    }
 
-   internal class DiscreteFormulaExport : FormulaExport
+   public class DiscreteFormulaExport : FormulaExport
```
Comment (msevestre) [inline src/OSPSuite.Core/Serialization/SimModel/DTO/FormulaExport.cs]: why this was internal

File: src/OSPSuite.Core/Serialization/SimModel/Services/CreateExportModelVisitor.cs [hunk 3]
```diff
@@ -282,49 +282,48 @@ private ExplicitFormulaExport mapProcessFormula(IProcess process)
          //resolve to ensure that mapping works as expected
          amountKinetic.ResolveObjectPathsFor(process);
 
-         return mapFormula(process, amountKinetic).DowncastTo<ExplicitFormulaExport>(); 
+         return mapFormula(process, amountKinetic).DowncastTo<ExplicitFormulaExport>();
       }
 
       private FormulaExport mapFormula(IUsingFormula usingFormula, IFormula formula)
       {
-         FormulaExport formulaExport;
```
Comment (msevestre) [inline src/OSPSuite.Core/Serialization/SimModel/Services/CreateExportModelVisitor.cs]: Little bit of a rewrite to make it easier on the eyes

**Comment by msevestre** [comment]
@Yuri05 Fix required in SimModel

File: src/OSPSuite.Core/Domain/Formulas/TableFormulaWithXArgument.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+      /// <summary>
+      ///    Return the value of the table object for 0-offset value
+      /// </summary>
+      protected override double CalculateFor(IEnumerable<IObjectReference> usedObjects, IUsingFormula dependentObject)
+      {
+         var tableObject = GetTableObject(dependentObject);
+         var tableFormula = tableObject?.Formula as TableFormula;
+
+         return tableFormula?.ValueAt(GetXArgumentObject(dependentObject).Value) ?? double.NaN;
+      }
```
Comment (Yuri05) [inline src/OSPSuite.Core/Domain/Formulas/TableFormulaWithXArgument.cs]: return NaN or better throw an Exception? (TableWithArgFormula uses a non-Table object)

File: src/OSPSuite.Core/Serialization/SimModel/Services/CreateExportModelVisitor.cs [hunk 5]
```diff
@@ -205,6 +203,7 @@ public void Visit(IReaction reaction)
          {
```
Comment (Yuri05) [inline src/OSPSuite.Core/Serialization/SimModel/Services/CreateExportModelVisitor.cs]: rea**c**tionPartner

Comment (Yuri05) [reply src/OSPSuite.Core/Serialization/SimModel/Services/CreateExportModelVisitor.cs]: is `reationPartner` in code (typo)

File: src/OSPSuite.Core/Domain/Formulas/TableFormulaWithXArgument.cs (hunk 4 above)
Comment (msevestre) [reply src/OSPSuite.Core/Domain/Formulas/TableFormulaWithXArgument.cs]: I am in favor of NaN but I can change if you prefer an exception

Comment (msevestre) [reply src/OSPSuite.Core/Domain/Formulas/TableFormulaWithXArgument.cs]: I am in favor of NaN but I can change if you prefer an exception

**Comment by msevestre** [comment]
This is old code that was not changed with this PR (at all). Please review
only code relevant to PR if possible.

**Comment by msevestre** [comment]
Going to wait for SimModel namespace update to update requested changes

**Comment by msevestre** [comment]
@Yuri05 This should be good to go. I will merge once CI reports success

**Pull request merged**